api/users/utils/mailer_lite.py

```diff
@@ -55,9 +55,10 @@ def _update_organisation_users(self, organisation_id: int):
 class BatchSubscribe(MailerLiteBaseClient, AbstractContextManager):
     resource = "batch"
 
-    def __init__(self):
+    def __init__(self, max_batch_size: int = DEFAULT_MAX_BATCH_SIZE):
```

Comment:
I don't think we should allow the user to set the limit(at least not without making sure that it's less than 50)? 
>There is a limit of maximum 50 requests per single batch.
from: https://developers.mailerlite.com/reference/batch-api-requests

Reply:
Ok, yep, good spot. I've just added a check in the __init__ method for now.